charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -29,6 +29,10 @@ sidecars:
     repository: k8s.gcr.io/sig-storage/csi-node-driver-registrar
     tag: "v2.0.1"
 
+snapshotController:
+  image: k8s.gcr.io/sig-storage/snapshot-controller
```

Comment:
should say "repository:" instaed of "image:", to beconsistent with the other values

Reply:
done